source/containers.tex

```diff
@@ -6946,7 +6946,7 @@
 \begin{itemdescr}
 \pnum
 \effects
-Constructs an empty set using the specified comparison objects and allocator.
+Constructs an empty \tcode{set} using the specified comparison objects and allocator.
```

Comment:
Well, the second std::set constructor in [set.cons] /does/ have the \tcode. I was merely trying to make it consistent.